### PR TITLE
Sfdk: Add the 'scrape' command. JB#59443

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

File: share/qtcreator/sfdk/modules/20-building-mb2/doc/command.format-patch.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+1) the sources are under version control by Git,
+2) another source repository is embedded as a Git submodule, and
+3) changes to the embedded repository may not be shared natively.
+
+In this case the changes to the embedded repository (submodule) may be shared in form of patch files versioned under the embedding repository (superproject). This command automates the job of collecting patches for new commits appearing in all (sub-)submodules and updating existing patch files in a safe manner.
+
+This command may be used in pair with the 'prepare' command, which is capable of doing the opposite job: apply patches as commits (unless the 'no-vcs-apply' configuration option is used).
+
+This command enforces certain patch formatting and file naming conventions. It is not guarranteed (not desired) that it produces patch files matching the original patch files applied with the help of the 'prepare' command.  Namely the file numbering is enforced, using discrete number ranges for patches originating from different submodules.
```
Comment (vigejolla) [inline share/qtcreator/sfdk/modules/20-building-mb2/doc/command.format-patch.adoc:11]: same type as on the mb2 help

**Review by vigejolla — APPROVED** [review]
one typo, otherwise LGTM

File: share/qtcreator/sfdk/modules/20-building-mb2/doc/command.format-patch.adoc (hunk 1 above)
Comment (martyone) [reply share/qtcreator/sfdk/modules/20-building-mb2/doc/command.format-patch.adoc]: Fixed

**Comment by martyone** [comment]
Synced after polishing more on mb2 side

**Review by vigejolla — APPROVED** [review]

**Pull request merged**